src/client/app/components/MultiSelectComponent.jsx

```diff
@@ -14,7 +14,7 @@ export default class MultiSelectComponent extends React.Component {
 		this.onValuesChangeInternal = this.onValuesChangeInternal.bind(this);
 		// selectedOptions holds a list of the options that have been selected
 		this.state = {
-			selectedOptions: this.props.selectedOptions ? this.props.selectedOptions : []
+			selectedOptions: this.props.selectedOptions ? this.props.selectedOptions : null
```

Comment:
Why are you using null here? That seems error-prone.

Reply:
This avoid a warning from react-bootstrap's select component about prop-types.

Reply:
This is null here because the adminComponent passes object to MultiSelect Component, not array.